numpy/core/tests/test_item_selection.py

```diff
@@ -79,9 +79,63 @@ def test_empty_partition(self):
         assert_array_equal(a, a_original)
 
     def test_empty_argpartition(self):
-            # In reference to github issue #6530
-            a = np.array([0, 2, 4, 6, 8, 10])
-            a = a.argpartition(np.array([], dtype=np.int16))
+        # In reference to github issue #6530
+        a = np.array([0, 2, 4, 6, 8, 10])
+        a = a.argpartition(np.array([], dtype=np.int16))
+
+        b = np.array([0, 1, 2, 3, 4, 5])
+        assert_array_equal(a, b)
+
+    def test_nonzero_safety(self):
```

Comment:
Though I can see why you put it here, this test should go in `core/tests/test_numeric.py:TestNonzero` so it's with the others